src/main/proto/de/cyface/protos/model/measurement.proto

```diff
@@ -656,8 +668,7 @@ message File {
    *
    * Using `64bit` as the timestamp is absolute and in milliseconds since 1970 and, thus, not in the `integer` range.
    *
-   * The value is encoded in the `uint` ProtoBuf format as we don't expect negative timestamps.
-   * Negative numbers are supported but encoded inefficiently in `uint` this data type, compared to `sint`.
+   * The value is encoded in the `uint` ProtoBuf format as we don't expect (and support) negative absolute timestamps.
    */
   uint64 timestamp = 16;
```

Comment:
same as in `Events`  --> non `releated` --> we only store `absolute` timestamps.